doc/method/method-intro.md

```diff
@@ -87,35 +76,40 @@ The combination S + S' is formed in two steps:
  1. A mapping or _alignment_ step that identifies all
     nodes in S' that can be equated with nodes in S. There will often be nodes
     in S' that cannot be aligned to S.
- 2. A _merge_ step that creates the union, U = S + S', by adding to S the unaligned
-    taxa from S'. The attachment position of unaligned nodes from step 1 is
+ 2. A _merge_ step that creates the combination U = S + S', by adding to S the unaligned
```

Comment:
I was trying to stick to the idea that the workspace is a data structure that *holds* a taxonomy, so U is a taxonomy that could be read out of a workspace, just as a woodshop might *hold* a chest of drawers without *being* one. + is an operator that yields a taxonomy, not a workspace. Will see if I can make this clearer without making it intrusive, because it's not very interesting.